new-pipeline/dist.html

```diff
@@ -159,29 +159,19 @@ <h3><select id="selected-key4" class="multiselect" title="Selected key"></select
                                     </select> from highest to lowest
                                 </div>
                                 <div>
-                                    <b>Table mode:</b> show histograms as tables of values rather than visual charts:
-                                    <label><input name="table-toggle" type="radio" value="0"> Show histograms</label>
-                                    <label><input name="table-toggle" type="radio" value="1"> Show tables</label>
+                                    <label><input name="table-toggle" type="checkbox" value="0"> <b>Table mode</b></label> Show histograms as tables of values rather than visual charts.
```

Comment:
This line is a little long now. Can we put the description on a new line?
Similarly for the elements below.